ckanext/canada/schemas/presets.yaml

```diff
@@ -3842,6 +3842,10 @@ presets:
         en: Indonesian
         fr: indonésien
       value: id
+    - label:
+        en: Innu-Aimun
+        fr: Innu-Aimun
+      value: innuaimun
```

Comment:
@RabiaSajjad I think we are using ISO codes for the values. https://iso639-3.sil.org/code/moe <- soe should be `moe` I guess.